hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestDynamoDBMetadataStore.java

```diff
@@ -1108,6 +1112,25 @@ private Table verifyTableInitialized(String tableName, DynamoDB dynamoDB) {
     return table;
   }
 
+  /**
+   * Verify the table is created with correct server side encryption (SSE).
+   */
+  private void verifyTableSse(Configuration conf, TableDescription td) {
+    SSEDescription sseDescription = td.getSSEDescription();
+    if (conf.getBoolean(S3GUARD_DDB_TABLE_SSE_ENABLED, false)) {
```

Comment:
```S3GUARD_DDB_TABLE_SSE_ENABLED``` should be true here, shouldn't it?
```assertEquals("ENABLED", sseDescription.getStatus());``` this will be only satisfied if the SSE is enabled?
Please correct me if I'm wrong.

Reply:
No this config will not be always true, since it depends on how you configure your test xml before running those tests. By default they are false, in which case we assert the `sseDescription` retruns `DISABLED`. But if you change your test config per the above `s3guard.md` doc, then this will be true and we then asset the `sseDescription` with `ENABLED` status and `KMS` SSE type.

We do not test key ARN is the same as configured value, because in configuration, the ARN can be specified by alias. But it is possible actually if we really want to test that, which will be more KMS code here.

Reply:
OK. Should we tweak testing.md to mention this

(FWIW, I think it's time for a reorg to list all options in one place, but not in this PR)